README

```diff
@@ -107,14 +107,14 @@ lib/include directory. There are also test programs in the test/
 directory which give examples of use.
 
 Python bindings are in bindings/python and are commented in a similar
-manner. There is also test code in the test/ directory which give
+manner. There is also a test code in the test/ directory which gives
```

Comment:
https://books.google.co.uk/books?id=kknpPZjUR5IC&pg=PA321&lpg=PA321&dq=%22give+examples+of+use%22&source=bl&ots=EmY0h3IaIW&sig=MQGBfjFNhMSI0RBgdipdYTEyWUM&hl=en&sa=X&ved=0ahUKEwi7kbTZzrvMAhWB2BoKHbBjD6sQ6AEILDAD#v=onepage&q=%22give%20examples%20of%20use%22&f=false

seems like a good enough example of its correctness